api/test/e2e/scenario-interventions/scenario-interventions.spec.ts

```diff
@@ -72,22 +97,108 @@ describe('ScenarioInterventionsModule (e2e)', () => {
 
       expect(response).toHaveJSONAPIAttributes(expectedJSONAPIAttributes);
     });
+  });
 
+  describe('Missing data and validations', () => {
     test('Create a scenario intervention without the required fields should fail with a 400 error', async () => {
       const response = await request(app.getHttpServer())
         .post('/api/v1/scenario-interventions')
         .set('Authorization', `Bearer ${jwtToken}`)
-        .send()
-        .expect(HttpStatus.BAD_REQUEST);
+        .send();
 
+      expect(HttpStatus.BAD_REQUEST);
       expect(response).toHaveErrorMessage(
         HttpStatus.BAD_REQUEST,
         'Bad Request Exception',
         [
-          'title should not be empty',
           'title must be shorter than or equal to 40 characters',
           'title must be longer than or equal to 2 characters',
+          'title should not be empty',
           'title must be a string',
+          'type must be a valid enum value',
+          'type should not be empty',
+          'type must be a string',
+          'startYear should not be empty',
+          'startYear must be a number conforming to the specified constraints',
+          'percentage should not be empty',
+          'percentage must be a number conforming to the specified constraints',
+          'scenarioId should not be empty',
+          'scenarioId must be a UUID',
+          'materialsIds should not be empty',
+          'each value in materialsIds must be a UUID',
+          'businessUnitsIds should not be empty',
+          'each value in businessUnitsIds must be a UUID',
+          'suppliersIds should not be empty',
+          'each value in suppliersIds must be a UUID',
+          'newIndicatorCoefficients must be a json string',
+          'newIndicatorCoefficients should not be empty',
+          'newIndicatorCoefficients must be a string',
+        ],
+      );
+    });
+
+    test('Create new intervention with replacing supplier without new location data fields should fail with a 400 error', async () => {
+      const scenario: Scenario = await createScenario();
+      const material: Material = await createMaterial();
+      const supplier: Supplier = await createSupplier();
+      const businessUnit: BusinessUnit = await createBusinessUnit();
+      const response = await request(app.getHttpServer())
+        .post('/api/v1/scenario-interventions')
+        .set('Authorization', `Bearer ${jwtToken}`)
+        .send({
+          title: 'test scenario intervention',
+          startYear: 2025,
+          percentage: 50,
+          scenarioId: scenario.id,
+          materialsIds: [material.id],
```

Comment:
Shouldn't this param be:
```suggestion
          'materialsIds[]': [material.id],
```|
To match the approach in other filters (and the way nestjs parses arrays of string)